packages/web-app-preview/src/App.vue

```diff
@@ -190,23 +179,46 @@ export default defineComponent({
         query: { ...unref(route).query, ...query }
       })
     }
+    const isFileContentLoading = ref(true)
+
+    const triggerActiveFileDownload = () => {
+      if (isFileContentLoading.value) {
+        return
+      }
+
+      useDownloadFile().downloadFile(activeFilteredFile.value)
```

Comment:
while this probably works here, composables should be instantiated on top level in `setup()`